### PR TITLE
Use only a specified cipher suite

### PR DESCRIPTION
**Pull request by AniruddhaKanhere**
Description
-----------
This PR modifies the code so that only one cipher suite is chosen for SSL connection. namely - MBEDTLS_TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256.

Test Steps
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kstribrnAmzn** [comment]
Why do we want to force this to only use one cipher suite?

**Comment by AniruddhaKanhere** [comment]
> Why do we want to force this to only use one cipher suite?

This restricts the scope of the certification. mbedTLS supports around 200 cipher suites. But, to limit the scope of the certification, we have chosen to use only this cipher suite as it is widely used by web browsers and provides best of both worlds - computation complexity and security.

**Review by kstribrnAmzn — APPROVED** [review]

File: .DS_Store
Comment (joshzarr) [inline .DS_Store:1]: Can this file be deleted?

**Review by joshzarr — APPROVED** [review]
Approved with removing the unnecessary DS_Store file

File: .DS_Store
Comment (AniruddhaKanhere) [reply .DS_Store:1]: Yes we can. But I want to make that change along with formatting changes that I shall be doing shortly.

**Pull request merged**